src/bundles/BuyerDashboard/components/Dashboard/BuyerDashboard.css

```diff
@@ -37,6 +32,7 @@
 
 .filters {
   margin: 1.5rem 0 0;
+  width: 74.48861%
```

Comment:
That's precision! It was a number that was being used and I couldn't see that it was calculated anywhere, it worked though.